app/views/admin/_admin_nav.html.erb

```diff
@@ -7,6 +7,9 @@
     <%= span_if_current ts('Post AO3 News', key: 'header'), new_admin_post_path %>
   </li>
   <li>
+    <%= link_to t("Delete Post"),@admin_post, confirm: 'Are you sure?', method: :delete, class: 'action' %>
```

Comment:
Oh, sorry! When you asked about `ts` vs `t`, I thought you meant `ts("Delete Post")` vs `t('admin.admin_nav.delete', default: "Delete Post")` -- if we're using `t`, I think we're doing it with a key.

There's a space missing between the comma and `@admin_post`, and `class: 'action'` is unnecessary. "Are you sure you want to delete this news post?" would be a better confirmation message (it helps to remind users what they're doing) to use instead, and it should be translatable.